htdocs/user/param_ihm.php

```diff
@@ -214,6 +214,28 @@
 if (isModEnabled('ticket')) {
 	$tmparray['ticket/list.php?mainmenu=ticket&leftmenu='] = 'Tickets';
 }
+// add bookmarks to available landing pages
+if (!empty($conf->global->MAIN_USE_BOOKMARKS_FOR_LANDING_PAGES)) {
+	$sql = "SELECT b.rowid, b.fk_user, b.url, b.title,";
+
+	$sql .= " FROM ".MAIN_DB_PREFIX."bookmark as b";
+	$sql .= " WHERE 1=1";
+	$sql .= " AND b.entity IN (".getEntity('bookmark').")";
+	$sql .= " AND b.url NOT LIKE 'http%'";
+	if (!$object->admin) {
+		$sql .= " AND (b.fk_user = ".((int) $object->id)." OR b.fk_user is NULL OR b.fk_user = 0)";
+	}
+	$resql = $object->db->query($sql);
+	if ($resql) {
+		$i = 0;
+		while ($i < $object->db->num_rows($resql)) {
```

Comment:
Can you move the $object->db->num_rows($resql) out of the while. Do it before and sav it into $num_rows then use num_rows into the while.